LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2019 Of Pine Wood
+Copyright (c) 2020 Joe Dluzen
```

Comment:
If you fork it you can change the license ;) I will add you as a contributor in the readme when your changes are accepted.

Reply:
sure no problem. some projects handle these things differently.